src/recurrer.ts

```diff
@@ -1,111 +1,155 @@
-import _ from 'lodash'
+import * as _ from 'lodash'
+import { ADDITIONAL_UNIT, DAYS_IN_WEEK, HOUR_IN_DAY, MIN_IN_HOUR, MS_IN_SEC, NO_DIFF, SEC_IN_MIN } from './constants'
 
-// For producing recurring timestamps 
-export default class Recurrer {
-  async getDailyRecurringTimestamps(
-    startTimestamp: number,
-    numberRecurring: number,
-    hourOfDay: HourOfDay,
-  ): Promise<number[]> {
+// For producing recurring timestamps
+export class Recurrer {
+  /**
+   * Input starting timestamp in milliseconds along with desired hour of day.
+   * Output up to 24 daily recurring timestamps in milliseconds
+   * @param startTimestamp
+   * @param numberRecurring
+   * @param hourOfDay
+   * @returns daily recurring timestamps
+   */
+  getDailyRecurringTimestamps(startTimestamp: number, numberRecurring: number, hourOfDay: HourOfDay): number[] {
```

Comment:
IMO `startTimestamp` is confusing as it's really just being used for the starting day.  If I am reading correctly.  I think it might be easier to read if it was `startDate`.

Reply:
haha it is confusing. idk what to call it tho. so a normal sane dev might be like, here is our first timestamp that we would like to input. please create following timestamps based on this timestamp. 

However, some lazy devs might choose not to do this. they might just say, here is Date.now(), which isn't an hour timestamp, or could be in the past, or whatever. i use that inputted timestamp as a marker and find the first matching timestamp based on the other parameters they give me

Reply:
probably need a little more desc in the jsdoc as well

Reply:
I understand a little better now and `startTimestamp` makes sense for how it is being used.  I do worry if in thinking about the "lazy" developers we are actually making things more confusing.  Requiring an exact date (or an exact hour for `getHourlyRecurringTimestamps`) will remove any ambiguity for the expected outcome of these functions.